Week-4/Assignment 4-2/index.js

```diff
@@ -0,0 +1,17 @@
+var Person = function () {};
+Person.prototype.initialize = function (name, age) {
+  this.name = name;
+  this.age = age;
+};
+
+var Teacher = function () {};
+Teacher.prototype = new Person();
+
+Teacher.prototype.teach = function (subject) {
+  console.log(this.name + " is now teaching " + subject);
+  console.log(this.name + " is of age " + this.age);
```

Comment:
age is not required

Reply:
in guideline example, they mention about age so I just added